domains/jacobmalmberg.json

```
@@ -4,6 +4,7 @@
         "email": "jacob_ost@hotmail.com"
     },
     "records": {
-        "CNAME": "cname.vercel-dns.com"
+        "CNAME": "cname.vercel-dns.com",
+        "TXT": "google-site-verification=IGGOBkDDOAkrFU3c5a5dT5D5BFhKs_j4qzd4plZYI-E"
     }
```

```suggestion
"records": {
        "CNAME": "cname.vercel-dns.com",
        "TXT": "google-site-verification=IGGOBkDDOAkrFU3c5a5dT5D5BFhKs_j4qzd4plZYI-E"
},
"proxied": true
```
you can't combine CNAME with other records unless it was proxied.